tests/kafkatest/services/streams.py

```diff
@@ -303,12 +303,20 @@ def start_node(self, node):
 class StreamsSmokeTestBaseService(StreamsTestBaseService):
     """Base class for Streams Smoke Test services providing some common settings and functionality"""
 
-    def __init__(self, test_context, kafka, command):
+    def __init__(self, test_context, kafka, command, num_threads = 3):
```

Comment:
Could you clarify the motivation of modifying the system test here? By default StreamsSmokeTest client is configured with 3 threads.

Reply:
Stated in the caller code